app/code/community/Bolt/Boltpay/etc/system.xml

```diff
@@ -177,6 +177,16 @@
               <show_in_store>1</show_in_store>
             </selector_styles>
 
+            <allowed_button_by_custom_routes>
+                <label>Allowed Button By Custom Route(s)</label>
```

Comment:
@YevhenShBolt, Relabeling this to say: "Show Bolt Button on Pages By Custom Route(s)" is more clear to me what this setting does.

Can we update it?

Reply:
@mccottry i will update. Thanks.

Reply:
@mccottry updated

Reply:
I don't think it's updated?